packages/elements/src/number-field/__test__/number-field.step.test.js

```diff
@@ -220,4 +220,43 @@ describe('number-field/Step', function () {
       await expectValues(el, [-1, -2, -3, -4, -5], UP, -1.5);
     });
   });
+
+  describe('Support long press', function () {
+    let el;
+    const dispatchLongTapEvent = async (el, wait) => {
```

Comment:
How about default `wait` parameter at 1000ms?
```suggestion
    const dispatchLongTapEvent = async (el, wait = 1000) => {
```